### PR TITLE
[TWW] Initial support for Resto Shaman

### PR DESCRIPTION
**Pull request by BrocoliCosmique**
Basic reactivation of the spec for 11.0 and minor addition to the Prototype guide : added Reactive Warding text when applicable, healing display on the right side yet to be implemented

### CODE REVIEW

File: src/analysis/retail/shaman/restoration/CHANGELOG.tsx [hunk 1]
```diff
@@ -1,48 +1,14 @@
...
-  Fassbrause,
-  niseko, Seriousnes,
-  ToppleTheNun,
-  Vetyst,
-  Vohrr,
+  Seriousnes,
   Ypp,
 } from 'CONTRIBUTORS';
-import { SpellLink } from 'interface';
+//import { SpellLink } from 'interface';
```
Comment (kfinch) [inline src/analysis/retail/shaman/restoration/CHANGELOG.tsx]: Please just remove unneeded imports, they can be added back in when they're needed

File: tsconfig.json [hunk 2]
```diff
@@ -59,9 +59,7 @@
     "src/analysis/retail/hunter/beastmastery",
     "src/analysis/retail/hunter/marksmanship",
     "src/analysis/retail/hunter/survival",
-    "src/analysis/retail/hunter/shared",
-    "src/analysis/retail/shaman/restoration",
-    "src/analysis/retail/shaman/shared"
+    "src/analysis/retail/hunter/shared"
```
Comment (kfinch) [inline tsconfig.json]: Mistaken removal of shaman from tsconfig?

File: tsconfig.json [hunk 3]
```diff
@@ -80,7 +78,6 @@
     "./src/analysis/retail/warrior/fury/CONFIG.tsx",
     "./src/analysis/retail/hunter/beastmastery/CONFIG.tsx",
     "./src/analysis/retail/hunter/marksmanship/CONFIG.tsx",
-    "./src/analysis/retail/hunter/survival/CONFIG.tsx",
-    "./src/analysis/retail/shaman/restoration/CONFIG.tsx"
+    "./src/analysis/retail/hunter/survival/CONFIG.tsx"
```
Comment (kfinch) [inline tsconfig.json]: Mistaken removal of shaman from tsconfig?

**Review by kfinch — CHANGES_REQUESTED** [review]

File: tsconfig.json (hunk 2 above)
Comment (BrocoliCosmique) [reply tsconfig.json]: Saw that on another commit ( https://github.com/WoWAnalyzer/WoWAnalyzer/pull/6915/commits/a1588eea958555d7803f2aeb4ee9ff05f66cd676 ) and thought it was required as part of the reactivation of the spec. Doesn't look like it makes any difference however, so now I'm confused as to what exactly is "excluded" about these specs.

**Comment by BrocoliCosmique** [comment]
Thanks for the review @kfinch, the code has been updated according to your remarks.

File: tsconfig.json (hunk 2 above)
Comment (kfinch) [reply tsconfig.json]: You know that's a fair point I'm not sure what the deal is with this partial list of specs here. Still, it is not needed for reactivation and we shouldn't change it until we know why we're changing it.

**Review by kfinch — APPROVED** [review]

File: tsconfig.json (hunk 2 above)
Comment (emallson) [reply tsconfig.json]: If you're re-enabling a spec, it should be removed from tsconfig. The exclusion only applies to un-imported files in dev mode but it seems to do weird things with lazy loading (?) in production if you enable a spec and don't remove from tsconfig. We'll see if there are any unusual errors with resto shaman 😅

Comment (kfinch) [reply tsconfig.json]: Oh shit my bad I think I got mixed up on what this was - I guess we'll see what happens lmao

**Pull request merged**